RevenueCatUI/CustomerCenter/Views/ScrollViewSection.swift

```diff
@@ -54,12 +54,16 @@ struct PurchasesInformationSection: View {
     var body: some View {
         ScrollViewSection(title: title) {
             ForEach(items) { purchase in
+                // only a few active subscriptions
```

Comment:
Wouldn't it be possible to do `ForEach(items.enumated())` to get the index directly? 

I think that's going to be possible with Swift 6.2, but I may be wrong 🤔